NEWS.md

```diff
@@ -9,12 +9,11 @@
 * add option to save as read only recommended ([#201](https://github.com/ychps/openxlsx/issues/201))
 * fixed writing hyperlink formulas ([#200](https://github.com/ychps/openxlsx/issues/200))
 * `write.xlsx()` now throws an error if it doesn't have write permissions ([#190](https://github.com/ycphs/openxlsx/issues/190))
-* `write.xlsx()` now again uses the default of `overwrite = TRUE` for saving files ([#249](https://github.com/ycphs/openxlsx/issues/249))
```

Comment:
why did you remove this?

Reply:
By mistake probably.  Didn't notice this.